tincr/cad/util/util.tcl

```diff
@@ -328,6 +329,55 @@ proc ::tincr::print { args } {
     }
 }
 
+proc ::tincr::print_list { args } {
+    # Summary:
+    # Prints a list to specified channel with the specified header
+
+    # Argument Usage:
+    # [-newline] : Add a new line character after printing each element. 
+    #              If this is not specified, a space will be printed between each element 
+    # -channel : The output channel to print to
+    # -header : The header to print before the list (i.e. HEADER item1 item 2)
+    # print_list : The list to print
+
+    # Return Value: none
+
+    # Categories: xilinxtclstore, byu, tincr, util
+
+    # Notes:
+    # The default channel is "stdout".
+    
+    # Usage:
+    # tincr::print_list [-channel arg] [-header arg] [-newline] print_list
+    
+    set newline 0
+    set channel ""
+    set header ""
+    ::tincr::parse_args {channel header} {newline} {} {print_list} $args
```

Comment:
Nit: I'm not a big fan of the `::tincr::parse_args` method because it doesn't translate well into doxygen documentation (i.e. Doxygen only sees a single "args" parameter in the method declaration, so it only documents it). I have been slowly switching out `parse_args` for legit Tcl method declarations:
```
proc ::tincr::print_list { { newline 0} { channel "" } { header "" } } {
...
```